evolver/calibration/actions.py

```diff
@@ -6,20 +6,40 @@
 
 
 class CalibrationAction(ABC):
-    class UserInput(pydantic.BaseModel): ...
+    def __init__(self, name: str, description: str):
+        self.description = description
+        self.name = name
+
+    class UserInput(pydantic.BaseModel):
+        pass
 
     @abstractmethod
     def execute(self, state: Dict[str, Any], payload: UserInput) -> Dict[str, Any]:
+        """
+        Execute the calibration action.
+
+        This method should be implemented by subclasses to perform specific calibration actions.
+        It takes the current state of the calibration procedure, and an optional payload, it processes them and
+        returns the updated state.
+
+        The payload is a pydantic model that is validated in the dispatch method of the CalibrationProcedure.
+
+        Args:
+            state (Dict[str, Any]): The current state of the calibration process.
+            payload (UserInput): The user input required to perform the action.
+
+        Returns:
+            Dict[str, Any]: The updated state after performing the action.
+        """
         pass
 
 
 class DisplayInstructionAction(CalibrationAction):
     class UserInput(pydantic.BaseModel):
         pass
 
-    def __init__(self, description: str, name: str):
-        self.description = description
-        self.name = name
```

Comment:
since name and description are always required, bumped them into __init__ for CalibrationAction.